src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.sex !== 'f' && (
+      <p className="Person__age">I am 37</p>
+    )}
+
+    {person.isMarried ? (
+      <p className="Person__partner">
+        {`${person.partnerName} is my ${person.sex === 'f' ? 'husband' : 'wife'}`}
+      </p>
+    ) : (
+      <p className="Person__partner">I am not married</p>
```

Comment:
keep this condition in variable like
```const partner = sex === 'f' ? 'husband' : 'wife';``` and use it here

also in order to write less code...you can write p tag only one time and use ternary check inside it